### PR TITLE
[Java] Adding profile picture need rework

### PR DESCRIPTION
**Pull request by arifBurakDemiray**

### CODE REVIEW

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
revert url part

**Comment by arifBurakDemiray** [comment]
[[Java] Adding profile picture need rework](https://trello.com/c/huahE7mf/1002-java-adding-profile-picture-need-rework)

File: sdk-java/src/test/java/ly/count/sdk/java/internal/UserEditorTests.java [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+    public void setPicturePath_localPath() {
+        Countly.instance().init(TestUtils.getBaseConfig());
+        Countly.session().begin();
+        File imgFile = TestUtils.createFile(imgFileName);
+        //set profile picture url and commit it
+        Countly.instance().user().edit().setPicturePath(imgFile.getAbsolutePath()).commit();
+        validatePictureAndPath(imgFile.getAbsolutePath(), null);
+
+        Countly.session().end();
+        validatePictureInRQ("{\"picturePath\":\"" + imgFile.getAbsolutePath() + "\"}", imgFile.getAbsolutePath());
```
Comment (ArtursKadikis) [inline sdk-java/src/test/java/ly/count/sdk/java/internal/UserEditorTests.java:64]: picture path should not be set in the user profile body

Comment (ArtursKadikis) [reply sdk-java/src/test/java/ly/count/sdk/java/internal/UserEditorTests.java:64]: it should have the mandatory params but no user profile things

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
if picture bytes or the picture url is set to "null" or empty string, trying to indicate that the picture should be erased from the server side, we should be sending 'picture=NULL '

**Pull request merged**